CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to `servo/rust-bindgen`
+
+Hi! We'd love to have your contributions! If you want help or mentorship, reach
+out to us in a GitHub issue, or stop by #servo on irc.mozilla.org and introduce
+yourself.
+
+* [Code of Conduct](#coc)
+* [Filing an Issue](#issue)
+* [Building](#building)
+* [Testing](#tests)
+  * [Overview](#tests-overview)
+  * [Running All Tests](#tests-all)
+  * [Running a Single, Specific Test](#tests-one)
+  * [Authoring New Tests](#tests-new)
+
+## Code of Conduct <span id="coc"/>
+
+We abide by the [Rust Code of Conduct][coc] and ask that you do as well.
+
+[coc]: https://www.rust-lang.org/en-US/conduct.html
+
+## Filing an Issue <span id="issue"/>
+
+Think you've found a bug? File an issue! To help us understand and reproduce the
+issue, provide us with:
```

Comment:
`try to provide us`? Some issues like #67 are really tricky to minimize :)

Worth commenting here maybe that running with RUST_LOG=bindgen spits a log really helpful for debugging.